src/index.js

```diff
@@ -8,3 +10,32 @@ export {
 
 export * from './service';
 export * from './message';
+
+export default (config) => {
+  const internalLogger = new Logger(config);
+
+  return (label) => {
+    const labelledLogger = internalLogger.Label(label);
+
+    const logger = level => (...args) => {
+      let msg;
+
+      if (args.length === 2) {
```

Comment:
ah I didn't notice you've already replied.
so seems like we have agreed on this 😛